app.js

```diff
@@ -45,22 +70,49 @@ function getRandomHidingSpot() {
     const index = Math.floor(Math.random() * hidingPlaces.length);
 
     // use the random index above and the array of hidingPlaces to get a random hiding place string
-
+    const correctHidingPlace = `${hidingPlaces[index]}`;
```

Comment:
this totally works, but you could also just `return hidingPlaces[index]` and save yourself the line if you wanted.